index.html

```diff
@@ -10,15 +10,23 @@
 </head>
 <body>
     <div class="start">
-        <nav>
-            <ul class="nav-bar">
-                <li class="nav-desktop" id="navPortfolio"><a href="#projectTitle">Portfolio</a></li>
-                <li class="nav-desktop" id="navAbout"><a href="#about">About</a></li>
-                <li class="nav-desktop" id="navContact"><a href="#contact">Contact</a></li>
-                <li class="nav-mobile" id="home"><a href="#section-1">Welcome</a></li>
-                <li class="nav-mobile" id="navBurger"><a href="#"><i class="fa fa-navicon"></i></a></li>
-              </ul>
-        </nav>
+        <div class="nav-container">
+            <div class="navMenu">
+                <div class="nav-mobile">
+                    <h2 class="brand"><a href="#section-1">Welcome</a></h2>
+                    <button class="navToggle" id="navToggle"><i class="fas fa-bars"></i></button>
+                </div>
+                <nav>
+                    <ul class="nav-bar" id="nav-desktop">
+                        <li class="nav-desktop" id="navPortfolio"><a href="#projectTitle" class="nav-link">Portfolio</a></li>
+                        <li class="nav-desktop" id="navAbout"><a href="#about" class="nav-link">About</a></li>
+                        <li class="nav-desktop" id="navContact"><a href="#contact" class="nav-link">Contact</a></li>
+                      </ul>
+                </nav>
```

Comment:
You have done a great job with your mobile portfolio menu. However, according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md), you are supposed to stick as much as possible to the chosen [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). You are currently left with the following styling changes to make your project perfect and ready for approval:

- [ ] When you toggle to show the menu, the bars are supposed to be changed with an "X" according to the Figma design. Kindly fix this.

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/181084976-09b88e80-92e6-448b-a4fa-057972d6fa81.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/181085061-a5c6e4ff-5430-4eff-b0d8-f54136ce54be.png)

- [ ] The "Welcome" text is supposed to be hidden according to the chosen Figma design when the menu is opened. Kindly hide it. You can hide it with JavaScript when the menu is activated with something like ```logo.style.display = 'none';```
- [ ] I have also realized that your menu links are not positioned at the correct side of the screen as shown in the Figma design. You can use something like ```text-align: left;``` to align your nav items to the left side of the screen as shown in the Figma design.

**This is how your Figma design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/181086298-318187ee-8e9c-45c6-902b-9533ee131b5c.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/181086464-250e74a0-081f-4cbb-aa02-a7ebed2f3fde.png)